scss/_settings_colors.scss

```diff
@@ -14,7 +15,7 @@ $color-x-dark:           #000 !default;
 $color-negative:         #d73024 !default;
 $color-warning:          #f99b11 !default;
 $color-positive:         #0f8420 !default;
-$color-information:      #007aa6 !default;
+$color-information:      $color-highlight !default;
```

Comment:
After a discussion with @yaili it would be better to unlink `$color-information` from `$color-highlight`. If someone updates the `$color-highlight` as the colour for links then it would have the side effect of changing `$color-information`.